### ISSUE TITLE
Fetch image from a clipboard

### ISSUE THREAD

**Issue by FeodorFitsner** [flet-dev/flet#349]
### Discussed in https://github.com/flet-dev/flet/discussions/336

<div type='discussions-op-text'>

<sup>Originally posted by **jet10000** September 16, 2022</sup>
I want to get clipboard screenshot image, How to watch clipboard images changes?</div>

**Comment by FeodorFitsner** [comment]
Yep, looks like https://pub.dev/packages/pasteboard is the way to go.

**Comment by yutams** [comment]
We would be happy if you could give a higher priority to implementing this functionality in order to use the image-reading generative AI!

**Comment by lk-itween** [comment]
May I ask when it is expected that the feature will be implemented? I urgently need it and take screenshot of Controls!

**Comment by ndonkoHenri** [comment]
> I urgently need it and take screenshot of Controls!

It has now been [implemented](https://docs.flet.dev/services/clipboard/#images) and will be part of v0.80.6. (You can also get it from the latest [pre-release](https://pypi.org/project/flet/#history))
Concerning the taking of screenshots, see the already existing [`Screenshot`](https://docs.flet.dev/controls/screenshot) control.
